packages/volto/src/components/manage/Controlpanels/Addons/AddonItem.tsx

```diff
@@ -83,7 +112,8 @@ const AvailableItem: React.FC<AvailableAddonProps> = ({ addon, onInstall }) => {
     <GridListItem
       key={addon['@id']}
       className="addon-item"
-      aria-describedby={`addon-desc-${addon.id}`}
+      textValue={`${addon.title} ${addon.description}. ${intl.formatMessage({ id: 'Press Enter to install this addon' })}`}
```

Comment:
@deodorhunter you add the period which shows up strange if you don't have an addon. See the test result here https://github.com/plone/volto/pull/6345/commits/416059d6f1cbe1423d83a68249cada263504c4f4#diff-023c7e608404538c6858332e1915002b4c248049367176e7d0a52f69097794a2R65